draft-ietf-mls-protocol.md

```diff
@@ -1388,6 +1390,7 @@ struct {
     ContentType content_type;
     opaque sender_data_nonce<0..255>;
     opaque encrypted_sender_data<0..255>;
+    opaque authenticated_content[length_of_authenticated_content];
```

Comment:
`opaque` already encompass the length, I believe.

Reply:
I was inspired by
```
    opaque content[length_of_content];
```
above (in MLSCiphertextContent). Is that one also incorrect?

Reply:
This can just be the same definition as in the `MLSCiphertext` object.  The only reason for the `[]` notation w.r.t. content is the weird encoding of `MLSCiphertextContent`, which IIRC is about to get reverted.

Reply:
Done.
I also renamed it to authenticated_data instead of authenticated_content, in order to keep the naming the same. (I was in the pickle here, since the data struct is called 'MLSCiphertextConterntAAD' and then the field inside is called authenticated_data. I initially thought that calling it authenticated_content will be better, but I believe it's better to keep the same name all over the place).